lib/FrontendTool/ReferenceDependencies.cpp

```diff
@@ -129,25 +129,23 @@ bool swift::emitReferenceDependencies(DiagnosticEngine &diags,
                                       const FrontendOptions &opts) {
   assert(SF && "Cannot emit reference dependencies without a SourceFile");
 
+  ReferencedNameTracker *rnt = SF->getReferencedNameTracker();
```

Comment:
Looks like there's a variable called "tracker" down below that does the same thing.

Reply:
Yea, I'll rename "rnt" to "tracker" and remove the 2nd definition. (The enclosing function is so long it was easy for me to miss it.)